stable/slackin/templates/_helpers.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
```

Comment:
Might be worth updating to the following -- https://github.com/kubernetes/charts/blob/86a5197973b9dcc5593bb7dd26b4ca476e3509e4/stable/prometheus/templates/_helpers.tpl#L24